setup.py

```diff
@@ -61,7 +67,7 @@ def find_version(*file_paths):
     ],
     install_requires=[
         "appdirs",
-        "gdal",
+        gdal,
```

Comment:
@rolker What is the point to retrieve the version of the already installed gdal?